optimade/validator/utils.py

```diff
@@ -13,6 +13,7 @@
 """
 
 import time
+
```

Comment:
Is this necessary? I think it might be nice to use `isort` as a pre-commit hook? Then all import statements will be unified for all :)

Reply:
This is obviously just a typo, but yeah, we should probably start using isort. Will remove this diff

```suggestion
```